src/App.js

```diff
@@ -14,11 +14,97 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends Component {
+  state = {
+    selectedGoods: ['Jam'],
+  }
 
+  clearGood = () => {
+    this.setState({ selectedGoods: [] });
+  }
+
+  addGood = (good) => {
+    this.setState(({ selectedGoods }) => ({
+      selectedGoods: [...selectedGoods, good],
+    }));
+  }
+
+  deleteGood = (good) => {
+    this.setState(state => ({
+      selectedGoods: state.selectedGoods.filter(product => product !== good),
+    }));
+  }
+
+  setTitle = () => {
+    let titleStr = '';
+    const { selectedGoods } = this.state;
+    const arr = selectedGoods.slice(0, -1);
+
+    if (selectedGoods.length === 1) {
+      titleStr = `${selectedGoods} is sellect`;
+    }
+
+    if (selectedGoods.length > 1) {
+      // eslint-disable-next-line
+      titleStr += `${arr.join(', ')} and ${selectedGoods[selectedGoods.length - 1]} are selected`;
```

Comment:
```suggestion
      titleStr = `${arr.join(', ')} and ${selectedGoods[selectedGoods.length - 1]} are selected`;
```

З загального потоку трохи вибивається(**IMhO**). В двох інших випадках ти присвоюєш нове значення, а тут додаєш